recipes/simspace/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "simspace" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/simspace-{{ version }}.tar.gz
+  sha256: 780ec91460b1a86629c13ef4f7839186860ccb177c01daa181df9e46544b5767
+
+build:
+  noarch: python
```

Comment:
Please have a look at our docs for `noarch: python` packages:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

particularly the use of the `python_min` jinja variable.